Assets/Scripts/Models/Tile.cs

```diff
@@ -112,15 +112,14 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
-    public float MovementModifier;
+    public float MovementModifier { get; private set; }
```

Comment:
I agree if those conditions can be modable too. Once we do that we might not even need the MovementModifier, since the Modifier will come from the Tile Condition.